online-shop/src/app/app.module.ts

```diff
@@ -7,6 +7,7 @@ import { ProductDetailsComponent } from './product-details/product-details.compo
 import { ProductListComponent } from './product-list/product-list.component';
 import { HeaderComponent } from './header/header.component';
 import { ShoppingCartComponent } from './shopping-cart/shopping-cart.component';
+import {HttpClientModule} from "@angular/common/http";
```

Comment:
you can set the prettier to format the code on save, and a very good practice is to run the lint before committing changes, it will tell you what flaws you have in code